src/documents/Contexts.md

```diff
@@ -61,15 +52,106 @@ at a particular component location in the tree are the same, as described in the
 ## Where context names are looked for ##
 
 Context names listed in a configuration are searched for at instantiation time, sequentially,
-in three kinds of "scopes" or "environments".
+in two kinds of "scopes" or "environments".
 
 * Most immediately, context names are searched for in the tree of currently instantiating
 components, searching __upwards__ through the tree from the parent of the component where the context
 name is held.
-* Following this, a search is made in the *dynamic environment*, which is notionally a separate tree
-of components bound to the currently executing stack frame (this facility is deprecated and will be
-removed in a future version of the framework)
-* Finally, context names are sought in the *static environment*, which is a tree of components
-attached globally to the JS VM (or context within it) as a whole.
+* Following this, a search is made in the *resolver root*, which is notionally a separate tree
+of components which holds all components which has been instantiated with the grade `fluid.resolveRoot` anywhere in the
```

Comment:
This mention of fluid.resolveRoot should be a link to the relevant section and should be followed by "(described below)"